apps/marsha/templates/services/nginx/dc.yml.j2

```diff
@@ -75,8 +75,8 @@ spec:
         - name: marsha-v-static
           persistentVolumeClaim:
             claimName: marsha-pvc-static
-        {% if activate_http_basic_auth or marsha_activate_http_basic_auth -%}
+{% if activate_http_basic_auth or marsha_activate_http_basic_auth -%}
```

Comment:
I'm already doing it in my PR

Reply:
yes you're right it's a mistake. I will anyway need to rebase on yours.